cpp/src/join/join_kernels.cuh

```diff
@@ -66,6 +66,7 @@ __global__ void build_hash_table( multimap_type * const multi_map,
         // using the row hash value to determine the location in the 
         // hash map where the new pair should be inserted
         const auto insert_location = multi_map->insert( thrust::make_pair( row_hash_value, i ),
+                                                        0, 1, // single hash table partition
```

Comment:
If you do stick to this interface, perhaps these should be the last two arguments so the defaults can be used...

Reply:
this is removed with 1291316